homeassistant/components/simplepush/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "simplepush",
   "name": "Simplepush",
   "documentation": "https://www.home-assistant.io/integrations/simplepush",
-  "requirements": ["simplepush==1.1.4"],
+  "requirements": ["simplepush==2.1.0"],
```

Comment:
The library shouldn't use `asyncio.run` inside a method:
https://github.com/simplepush/simplepush-python/blob/72f97a53ba6bfd9b0a26634e8eadb9d83c3543e7/simplepush/__init__.py#L54

`asyncio.run` should only be called once for the asyncio application lifetime.

https://docs.python.org/3/library/asyncio-task.html#asyncio.run

Reply:
Please tag github releases corresponding to the PyPI releases to make it easier for the community to track changes. Thanks!

Reply:
Thank you for your review. I just released a new Python version that is tagged and doesn't use `asyncio.run`.

Reply:
I don't see any tags:
![image](https://user-images.githubusercontent.com/1858925/197180588-5bffe431-448a-4db7-a736-bfb96e50ee47.png)

You can still tag the version now as long as there wasn't any new commit.
Please also make sure to update the PR description to 2.1.1 instead of 2.1.0 and the git diff

Reply:
It's possible to tag old commits too. Just select the commit to tag when tagging.

Reply:
@thecode Thanks!
I forgot that I had to explicitly push the tag to Github.
It's on Github now and I changed the PR description.
Not sure which git diff you are talking about?

Reply:
I've updated the PR description. See the 3rd bullet with the link to the library for the diff.

Reply:
I added a tag for the old version (1.1.4) as well and adjusted the git diff.